OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -73,16 +73,28 @@ public void Created(Actor self)
 
 		public virtual void Tick(Actor self)
 		{
-			if (attack != null && !attack.IsAttacking)
+			// NOTE: FaceTarget is called in AttackTurreted.CanAttack if the turret has a target.
+			if (attack != null)
 			{
-				if (realignTick < info.RealignDelay)
-					realignTick++;
-				else if (info.RealignDelay > -1)
-					DesiredFacing = null;
+				if (!attack.IsAttacking)
+				{
+					if (realignTick < info.RealignDelay)
+						realignTick++;
+					else if (info.RealignDelay > -1)
+						DesiredFacing = null;
+
+					MoveTurret();
+				}
 			}
 			else
+			{
 				realignTick = 0;
+				MoveTurret();
+			}
+		}
 
+		private void MoveTurret()
```

Comment:
Explicit `private` should be avoided according to our https://github.com/OpenRA/OpenRA/wiki/Coding-Standard